### ISSUE TITLE
Hardcoded Graphics Being Background to the Players (1.0 to Build 59) on TheXTech

### ISSUE THREAD

**Issue by vitzie629** [Wohlstand/TheXTech#474]
While SMBX 1.0 to Build 59 had the logo & curtain hardcoded GFX act as backgrounds so players can be shown on the hardcoded GFX as shown below for example: SMBX 1.0 intro:
![ezgif-1-81915614ae](https://user-images.githubusercontent.com/90484844/214159194-8a0d59b5-15a2-43a1-8f4b-eb45a5a817ed.gif)

However, since SMBX 1.3, the hardcoded GFX are foreground to the players so players can be shown through the GFX as shown below: 
![Scr_2023-01-23_16-59-43](https://user-images.githubusercontent.com/90484844/214159468-3eacd53b-89d2-4dfb-ae1f-55e07fda023d.gif)

I feel that TheXTech should have an option where the hardcoded GFX can either be foreground (since SMBX 1.3) or background (SMBX 1.0 to Build 59).

**Comment by Wohlstand** [comment]
Good idea for the `gameinfo.ini` to specify the behaviour of the logo.

**Comment by vitzie629** [comment]
> Good idea for the `gameinfo.ini` to specify the behaviour of the logo.

Thank you! This can be useful for the nostalgic SMBX-Tech assets pack due to the intro level having the 1.0, 1.0.1, & 1.0.2 styled level!

**Comment by LoveBodhi** [comment]
> > Good idea for the `gameinfo.ini` to specify the behaviour of the logo.
> 
> Thank you! This can be useful for the nostalgic SMBX-Tech assets pack due to the intro level having the 1.0, 1.0.1, & 1.0.2 styled level!

Also SMB1 Styled Asset Packs / Levels.
